Dockerfile

```diff
@@ -39,8 +40,9 @@ WORKDIR /p4c/
 RUN apt-get update && \
     apt-get install -y --no-install-recommends $P4C_DEPS $P4C_RUNTIME_DEPS && \
     pip install tenjin && \
-    ./bootstrap.sh --enable-p4runtime-to-pd && \
+    mkdir build && \
     cd build && \
+    cmake .. && \
```

Comment:
Since writing this comment, some of these issues have been addressed: the patch now includes GTest support, and while it does regress compilation time, it speeds up the tests, resulting in the overall time required for a Travis run being about the same.

In the interests of allowing the state of the two build systems to drift apart, it'd make life easier to switch over to cmake as soon as possible. I'd still recommend that we don't do it as part of this PR, though. Let's land this, then open a new PR to make the changeover official (i.e., change the build system we use on Travis). We can wait to land that PR until everyone has had a chance to try the new build system out locally and see if any issues crop up.